aten/src/ATen/templates/TypeDefault.h

```diff
@@ -21,6 +21,9 @@ struct CAFFE2_API TypeDefault : public TypeExtendedInterface {
   bool is_sparse() const override {
     return backend() == Backend::SparseCPU || backend() == Backend::SparseCUDA || backend() == Backend::SparseHIP;
   }
+  bool is_quantized() const override {
+    return backend() == Backend::AffineCPU || backend() == Backend::PerChannelAffineCPU;
```

Comment:
Why can't we just use the data type to know whether a tensor `is_quantized`?

Reply:
we probably can, but I'm using backend() to keep consistent with other code like is_sparse as well.

Reply:
Hmm, CPU backend with `QInt8` datatype is not a quantized tensor?

Reply:
Yeah that is correct